client/app/scripts/charts/node.js

```diff
@@ -93,37 +83,37 @@ class Node extends React.Component {
           <div className={labelClassName}>
             <MatchedText text={label} match={matches.get('label')} />
           </div>
-          <div className={subLabelClassName}>
-            <MatchedText text={subLabel} match={matches.get('sublabel')} />
+          <div className={labelMinorClassName}>
+            <MatchedText text={labelMinor} match={matches.get('labelMinor')} />
           </div>
-          {!blurred && <MatchedResults matches={matchedNodeDetails} />}
+          <MatchedResults matches={matchedNodeDetails} />
         </div>
       </foreignObject>
     );
   }
 
   render() {
-    const { blurred, focused, highlighted, networks, pseudo, rank, label,
-      transform, exportingGraph, showingNetworks, stack } = this.props;
-    const { hovered, matched } = this.state;
+    const { focused, highlighted, networks, pseudo, rank, label, matches,
+      transform, exportingGraph, showingNetworks, stack, id, metric } = this.props;
+    const { hovered } = this.state;
 
     const color = getNodeColor(rank, label, pseudo);
     const truncate = !focused && !hovered;
     const labelOffsetY = (showingNetworks && networks) ? 40 : 28;
 
     const nodeClassName = classnames('node', {
+      // NOTE: Having a CSS animation here might not be the best idea.
+      // See https://github.com/weaveworks/scope/issues/2255
+      matched: !matches.isEmpty(),
```

Comment:
I'm getting an error where sometimes `matches` is undefined, which causes `matches.isEmpty()` to throw. Maybe check that it exists before calling `isEmpty`.